frontend/core/blocks/image-augmentations/computations.py

```diff
@@ -0,0 +1,354 @@
+# Python block template
+
+
+def compute(img_path):
+    '''Data Augmentation block for Computer Vision
+
+    Inputs:
+    img_path (str): image path
+    
+     
+    Outputs:
+    augmented_img_path (all): result images path
+    
+    '''
+
+    import albumentations as A
+    import cv2
+   
+    def rotate(input_image_numpy, parameter=10):
+        test_name = "rotate"
+        description = 'https://albumentations.ai/docs/api_reference/augmentations/geometric/rotate/#albumentations.augmentations.geometric.rotate.Rotate'
+        parameter = int(round(parameter))
+        transform = A.Compose([A.Rotate(limit=[parameter, parameter], interpolation=1, border_mode=0, value=0, mask_value=None, always_apply=True)])
+        transformed = transform(image=input_image_numpy)
+        transformed_image = transformed["image"]
+        return {'test_name': test_name, 'parameters': parameter, 'image': transformed_image, 'description': description}
+
+    def translate_horizontal(input_image_numpy, parameter=1):
+        test_name = "translate_horizontal"
+        description = ''
+        transform = A.Compose([A.ShiftScaleRotate(shift_limit=[0, 0], scale_limit=[0, 0], rotate_limit=[0, 0], interpolation=1, border_mode=0, value=0, mask_value=None, shift_limit_x=[parameter, parameter], shift_limit_y=None, always_apply=True)])
+        transformed = transform(image=input_image_numpy)
+        transformed_image = transformed["image"]
+        return {'test_name': test_name, 'parameters': parameter, 'image': transformed_image, 'description': description}
+
+    def translate_vertical(input_image_numpy, parameter=1):
+        test_name = "translate_vertical"
+        description = ''
+        transform = A.Compose([A.ShiftScaleRotate(shift_limit=[0, 0], scale_limit=[0, 0], rotate_limit=[0, 0], interpolation=1, border_mode=0, value=0, mask_value=None, shift_limit_x=None, shift_limit_y=[parameter, parameter], always_apply=True)])
```

Comment:
parameters are hardcoded is that okay or will there be some need for flexibility for the parameters